portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/SiteHandler.java

```diff
@@ -589,7 +589,47 @@ public void doSite(HttpServletRequest req, HttpServletResponse res, Session sess
 
 		//Show a confirm dialog when publishing an unpublished site.
 		rcontext.put("publishSiteDialogEnabled", ServerConfigurationService.getBoolean("portal.publish.site.confirm.enabled", false));
-
+		for (SitePage pageNow: site.getPages()){	//build Manage link for site access
+			if(pageNow.getTools().get(0).getToolId().equals("sakai.siteinfo")){
+				rcontext.put("manageurl", pageNow.getUrl() + "?sakai_action=doMenu_edit_access");
+				break;
+			}
+		}
+		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+		SimpleDateFormat readableDate = new SimpleDateFormat("MM/dd/yyyy hh:mm aa");
+		SimpleDateFormat readableTermDate = new SimpleDateFormat("MMMM dd, yyyy");
```

Comment:
We should be using UserTimeService instead as it will localize these dates for us